Description: Fix some minor issues
Author: Guilhem Bonnefille <guilhem.bonnefille@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- python-cloudfiles-1.7.11.orig/cloudfiles/consts.py
+++ python-cloudfiles-1.7.11/cloudfiles/consts.py
@@ -1,6 +1,6 @@
 """ See COPYING for license information. """
 
-__version__ = "1.7.10"
+__version__ = "1.7.11"
 user_agent = "python-cloudfiles/%s" % __version__
 us_authurl = 'https://auth.api.rackspacecloud.com/v1.0'
 uk_authurl = 'https://lon.auth.api.rackspacecloud.com/v1.0'
--- python-cloudfiles-1.7.11.orig/cloudfiles/__init__.py
+++ python-cloudfiles-1.7.11/cloudfiles/__init__.py
@@ -77,12 +77,17 @@ Set the logs retention on CDN-enabled/pu
 See COPYING for license information.
 """
 
+import warnings
+
 from cloudfiles.connection     import Connection, ConnectionPool
 from cloudfiles.container      import Container
 from cloudfiles.storage_object import Object
 from cloudfiles.consts         import __version__, \
 us_authurl, uk_authurl
 
+warnings.warn("python-cloudfiles has been deprecated as of August 1, 2013. "
+              "Please see https://github.com/openstack/python-swiftclient.",
+              DeprecationWarning)
 
 def get_connection(*args, **kwargs):
     """
--- python-cloudfiles-1.7.11.orig/setup.py
+++ python-cloudfiles-1.7.11/setup.py
@@ -1,12 +1,20 @@
 #!/usr/bin/python
+
+import warnings
+
 from setuptools import setup, find_packages
 from cloudfiles.consts import __version__
 
+warnings.simplefilter('default')
+warnings.warn("python-cloudfiles has been deprecated as of August 1, 2013. "
+              "Please see https://github.com/openstack/python-swiftclient.",
+              DeprecationWarning)
+
 setup(name='python-cloudfiles',
       version=__version__,
       description='CloudFiles client library for Python',
       classifiers=[
-        'Development Status :: 5 - Production/Stable',
+        'Development Status :: 7 - Inactive',
         'Environment :: Console',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
 # vim:set ai ts=4 sw=4 tw=0 expandtab:
